source/administration/object-management/transition-objects-to-s3.rst

```diff
@@ -178,28 +188,20 @@ The example above uses the following arguments:
        source MinIO deployment to faciliate ease of operations related to
        diagnostics, maintenance, or disaster recovery.
 
-   * - :mc-cmd:`ACCESS_KEY <mc admin tier add --access-key>`
-     - The S3 access key MinIO uses to access the bucket. The
-       access key *must* correspond to an IAM user with the 
-       required 
-       :ref:`permissions 
-       <minio-lifecycle-management-transition-to-s3-permissions-remote>`.
+   * - :mc-cmd:`STORAGE_CLASS <mc ilm tier add --storage-class>`
+     - The S3 storage class to which MinIO transitions objects. Specify
+       one of the following supported storage classes:
 
-   * - :mc-cmd:`SECRET_KEY <mc admin tier add --secret-key>`
-     - The corresponding secret key for the specified ``ACCESS_KEY``.
+       - ``STANDARD``
+       - ``REDUCED``
 
-   * - :mc-cmd:`REGION <mc admin tier add --region>`
+   * - :mc-cmd:`REGION <mc ilm tier add --region>`
      - The AWS S3 region of the specified ``BUCKET``. You can safely omit this
        option if the ``HOSTNAME`` includes the region.
 
-   * - :mc-cmd:`STORAGE_CLASS <mc admin tier add --storage-class>`
-     - The S3 storage class to which MinIO transitions objects. Specify
-       one of the following supported storage classes:
 
-       - ``STANDARD``
-       - ``REDUCED``
 
-3) Create and Apply the Transition Rule
+1) Create and Apply the Transition Rule
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
A thought - should we drop the references to `STORAGE_CLASS`? It's rare enough of an option for users to set, and has the risk of someone transitioning objects to MinIO w/ lowered EC, which is a risk.

Reply:
Hmm ... I'd lean to describing anything that shows up in the CLI help myself.